packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -2164,6 +2178,18 @@ cp1Rusd_rhedgeParentConfig_collateralAddress = "<%= contracts.RUSDProxy.address
 cp1Rusd_rhedgeParentConfig_priceHaircutUnscaled = "0.075"
 cp1Rusd_rhedgeParentConfig_autoExchangeDiscountUnscaled = "0.005"
 cp1Rusd_rhedgeParentConfig_oracleNodeId = "<%= settings.hedgeUsdcReyaLmNodeId %>"
+# -----------------------------------------------
+# TODO later: pass depositingEnabled through var (but there's no parseBool)
+# cp1Rusd_rusd_depositingEnabled = "true"
+cp1Rusd_wsteth_capUnscaled = "10"                                                   # todo: configure
+cp1Rusd_wsteth_autoExchangeThresholdUnscaled = "0"
```

Comment:
these values should be aligned with aa as well right?

cp1Rusd_wsteth_autoExchangeInsuranceFeeUnscaled
and 
cp1Rusd_wsteth_autoExchangeThresholdUnscaled

Reply:
right